scripts/deploy/govcms-db-sync

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
+IFS=$'\n\t'
+set -euo pipefail
+
+#
+# GovCMS database sync
+#
+# This will pull a database from the production
+# environment.
+#
+
+LAGOON_ENVIRONMENT_TYPE=${LAGOON_ENVIRONMENT_TYPE:-production}
+GOVCMS_DEPLOY_WORKFLOW_CONTENT=${GOVCMS_DEPLOY_WORKFLOW_CONTENT:-retain}
+GOVCMS_SITE_ALIAS=${GOVCMS_SITE_ALIAS:-govcms.prod}
+
+echo "GovCMS Deploy :: Database synchronisation"
+
+if [ "$LAGOON_ENVIRONMENT_TYPE" = "production" ]; then
+  echo "[skip]: Production environment can't be synced."
+  exit 0
+fi
+
+echo "[info]: Check that the site can be bootstrapped"
+
+if ! drush status --fields=bootstrap | grep -q "Successful"; then
+  # This will ensure that the databse is synchronised on the first deploy
+  # of an enviornment in lagoon.
+  echo "[info]: Site could not be bootstrapped... syncing."
+  drush --alias-path=/etc/drush/sites sql-sync @"$GOVCMS_SITE_ALIAS" @self -y
```

Comment:
could you please confirm that this alias path will play nicely with all platform types (SaaS and PaaS). Just in case if the location of the alias is different.